index.d.ts

```diff
@@ -249,3 +249,17 @@ console.log(hexToUint8Array('48656c6c6f'));
 ```
 */
 export function hexToUint8Array(hexString: string): Uint8Array;
+
+/**
+Get up to a 48-bit integer from a DataView`
+
+Replacement for [`Buffer#readUintBE`](https://nodejs.org/api/buffer.html#bufreadintbeoffset-bytelength)
+*/
+export function getUintBE(view: DataView): number; // eslint-disable-line @typescript-eslint/naming-convention
+
+/**
+ * Find a sequence of bytes in an array buffer
+ *
+ * Replacement for [`Buffer#indexOf`](https://nodejs.org/api/buffer.html#bufindexofvalue-byteoffset-encoding)
+ */
+export function findSequence(arrayBuffer: ArrayBuffer, sequence: Uint8Array): number;
```

Comment:
Why not call it `indexOf`? The current naming is confusing as `findSequence` makes it sound like it returns the sequence it finds.

Reply:
You're right, `find` implies that it returns the sequence. Changing it to `indexOf`